chalicelib/wrangler_checks.py

```diff
@@ -11,9 +11,90 @@
 import sys
 import json
 import datetime
+import time
 import boto3
 
 
+@check_function()
+def biorxiv_is_now_published(connection, **kwargs):
+    check = init_check_res(connection, 'biorxiv_is_now_published')
+    chkstatus = ''
+    chkdesc = ''
+    # run the check
```

Comment:
I don't think it's an immediate issue since there aren't many publications, but the `limit=all` search could be dangerous at some point. Maybe add `frame=object` to the url to keep the request size down? Not high priority because I will go through foursight at some point and replace all `limit=all` cases with a safe search.

Reply:
Followed what was used in the hi_glass url check (which BTW seems much more dangerous given the number of processed files but would be happy to switch to safe_search - we maybe should have a shared safe search function then could add specific callback?